index.html

```diff
@@ -84,14 +84,159 @@ <h1>Hey there.<span class="span__br"><br /></span> I’m Mateo.
             </ul>
           </div>
 
-      <div class="arrow_button">
-        <a href="#">
-          <img src="./assets/button/arrowbutton.svg">
-        </a>
+        <div class="arrow_button">
+          <a href="#">
+            <img src="./assets/button/arrowbutton.svg">
+          </a>
+        </div>
       </div>
     </div>
+  </section>
+
+  <!----Projects Section---------->
+
+  <section id="projects">
+    <div class="projects">
+      <h2 class="project_title">
+        My Recent <span class="span__br"><br /></span>Works
+      </h2>
+      <div class="projects_h2_icon"></div>
+      <div class="project_wrapper"></div>
+        <article class="project_card">
+          <div class="project_card_wrapper">
+            <h4 class="project_card_title">Multi-Post Stories Gain+Glory</h4>
+            <ul class="project_card_tags">
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+
+            <button class="project_card_button" type="button">See project</button>
+              
+          </div>
+        </article>
+
+        <div class="project_wrapper"></div>
+        <article class="project_card">
+          <div class="project_card_wrapper">
+            <h4 class="project_card_title">Multi-Post Stories Gain+Glory</h4>
+            <ul class="project_card_tags">
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+
+            <button class="project_card_button" type="button">See project</button>
+              
+          </div>
+        </article>
+
+        <div class="project_wrapper"></div>
+        <article class="project_card">
+          <div class="project_card_wrapper">
+            <h4 class="project_card_title">Multi-Post Stories Gain+Glory</h4>
+            <ul class="project_card_tags">
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+
+            <button class="project_card_button" type="button">See project</button>
+              
+          </div>
+        </article>
```

Comment:
- [ ] [Optional] Great job on the UI design! 👍 However, there is a requirement in the project to use CSS Grid for the main container that contains all of the cards, and Flexbox to lay out an individual card. Currently, it seems that your project is missing the container with the grid layout. Make sure to wrap your project elements in a container that utilizes CSS Grid to align the cards properly. This will ensure that your project follows the design requirements. Keep up the good work! 💪